index.rst

```diff
@@ -44,14 +44,636 @@
+With moderate effort, the functionalities of this tool can be expanded to to meet the requirements for on-the-fly applications.
+
+The working group has performed several proof-of-concept demonstrators to test and back our recommendations and findings.
+Critical to the success of using these tools in an integrated way will be the creation of a series of instructional documentation to guide users on how to design, communicate, build, and deploy their desired tools.
+The creation of these instructive documents is beyond the scope of this working group, but all members are keen to assist in developing this documentation.
+
+Lastly, this working group has accumulated numerous findings and has made recommendations that are critical to the success of commissioning, but technically outside the scope of the on-the-fly displays and analysis, as per the charge.
+These findings should be strongly considered by Project Management as they are expected to result in simplifying some of the global display challenges, specifically during data analysis.
+
+
+Introduction
+============
+
+The First-Look Analysis and Feedback Functionality (FAFF) Breakout Group met approximately weekly starting June 30, 2021 and released it's first report on January 14, 2022.
+The report consists of the use-cases that were used to define the functionality, requirements put on by those use-cases, and recommendations for new functionality.
+We created demonstrations of new functionality in several cases, which lengthened the response process, but also resulted in the building of proof-of-concept tools which enhances the viability of the recommendations.
+
+Some of the responses to the `charge`_ of the FAFF group were built off an Image Display Working Group who reported what was required by the data management group for image display and manipulation capabilities in `DMTN-126`_.
+That group was not focused on on-the-fly functionality but many of the findings were applicable and taken into account here when it made sense to do so.
+
+This report is structured into sections, where the first addresses each of the charge questions individually.
+This is followed by a section on findings that are pertinent to the commissioning team but fall outside the scope of the charge, and subsequently a section outlining the proof-of-concept demonstrators that helped inform our recommendations.
+
+
+Responses to Charge Questions and Deliverables
+==============================================
+
+The use-cases referenced throughout the document are all found on the :ref:`pg-D1-Use-cases` page.
+In each of the subsections
+
+.. _Deliverable 1:
+
+Deliverable 1: Use-Cases
+------------------------
+
+.. note:: 
+
+   The deliverable description from the `charge`_ is in the note below to ease readability 
+
+   1. A series of use-cases where feedback to observers is required. A reduced set of use-cases should be created as a regular reference during design and development.
```

Comment:
line break: 

> observers is required.
> A reduced set of

Reply:
Something about adding a line break in a numbered series was causing my local (VSCode extension) formatter to do weird things so I stopped doing it. I'm not sure if it's the extension or a sphinx thing but with my current machine diagnosing it is really time consuming :(

Reply:
I think you need to align the indentation, e.g.;

```
    1. A series of use-cases where feedback to observers is required.
       A reduced set of use-cases should be created as a regular reference during design and development.
```